app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -72,6 +96,11 @@ public View getChildView(int groupPosition, final int childPosition,
         return convertView;
     }
 
+    /**
+     * returns total children count.
```

Comment:
Can you please tell what comments did you find unnecessary so that I can remove them